Tests/Microsoft.AppCenter.Crashes.Test.Windows/CrashesTest.cs

```diff
@@ -201,5 +213,164 @@ public void ProcessPendingErrorsDoesNotCrashOnFileDeletionFailure()
             Mock.Get(ErrorLogHelper.Instance).Verify(instance => instance.InstanceRemoveStoredErrorLogFile(It.IsAny<Guid>()), Times.Never());
             Mock.Get(mockFile).Verify(file => file.Delete(), Times.Once());
         }
+
+        [TestMethod]
+        public void PlatformNotifyUserConfirmationDefault()
+        {
+            ErrorLogHelper.Instance = GenerateErrorLogHelperWithPendingFile();
+
+            // Start Crashes.
+            Crashes.SetEnabledAsync(true).Wait();
+            Crashes.Instance.OnChannelGroupReady(_mockChannelGroup.Object, string.Empty);
+            Crashes.Instance.ProcessPendingErrorsTask.Wait();
+
+            // Verify the value has been set.
+            var alwaysSendValue = AppCenter.Instance.ApplicationSettings.GetValue(Crashes.PrefKeyAlwaysSend, true);
```

Comment:
Naive question: Where is the code to initialize this value? And why is it necessary to do so in the first place?

Reply:
@achocron The second paramater of the `GetValue` is the `defaultValue`, so the `alwaysSendValue` will default to `true` if it hasn't been initialized